src/content/docs/telemetry-data-platform/manage-data/nrintegrationerror.mdx

```diff
@@ -0,0 +1,92 @@
+---
+title: Use NrIntegrationError event to understand data ingest problems
+metaDescription: Use the NrIntegrationError event to understand problems with ingest, integration with New Relic, and limits. 
+---
+
+Use New Relic's `NrIntegrationError` event to understand problems related to data ingest and feature configuration.  
+
+## What is NrIntegrationError used for? [#overview]
+
+Our `NrIntegrationError` event is used to record error messages related to: 
+
+* Exceeding data limits 
+* Malformed data
+* Configuration issues
+
+You can run queries to analyze the errors that are being generated. You can also set up alerts to be notified when specific types of errors, or quantities of errors, occur. 
+
+The primary way `NrIntegrationError` events are categorized is by the `newRelicFeature` attribute. `newRelicFeature` values include: 
+
+* `Metrics` indicates an issue with the Metric API or the tools that use that API.
+* `Event API` indicates an issue with the Event API. 
+* `Distributed Tracing` indicates an issue with distributed tracing (`Span`) data. 
+
+For attribute definitions, see the [data dictionary](/attribute-dictionary/?event=NrIntegrationError). 
+
+## Query and alert [#query-alert]
+
+You can query `NrIntegrationError` events using [NRQL](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language), and set up [NRQL alert conditions](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions) to get notified when issues happen. 
+
+Here's an example query that will show Event API ingest issues from the last week, with up to 100 results: 
+
+```
+SELECT message FROM NrIntegrationError WHERE newRelicFeature = 'Event API' SINCE 1 WEEK AGO LIMIT 100
+```
+
+Here's a similar query that shows issues with the Metric API: 
+
+```
+SELECT message FROM NrIntegrationError WHERE newRelicFeature = 'Metrics' SINCE 1 WEEK AGO LIMIT 100
+```
+
+Here's a similar query that shows issues with distributed tracing data: 
+
+```
+SELECT message FROM NrIntegrationError WHERE newRelicFeature = 'Distributed Tracing' SINCE 1 WEEK AGO LIMIT 100
+```
+
+Here's a query showing a count of events over time: 
+
+```
+SELECT count(*) FROM NrIntegrationError WHERE newRelicFeature = 'Metrics' TIMESERIES
+```
+
+If you want to see the structure of this event, you can use `keyset()`, like this: 
+
+```
+FROM NrIntegrationError SELECT keyset()
+```
+
+## Tutorials and examples [#use-cases]
+
+Here are links to docs that go into more detail about how `NrIntegrationError` can be used. 
```

Comment:
Where are the links? 

Also, many sentences in this page start with "Here are" or "Here is" - the repetition deemphasizes the thing that you're saying is there, and kind of makes the info boring. Maybe you can avoid the whole "here is" part and just identify the query or whatever thing you're talking about.